student-work/assignment-4/swayam.md

```
@@ -0,0 +1,11 @@
+#swayam
+
+Greetings I am Swayam and my favourite programming language is Java as I like how structured it was and how rigidly it abided by rules. It was also the first programming language that I learned.
+
+##Example code
+``` system.output.println('lol')
+```
+
+###Code Explanation
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)